aspnetcore/index.yml

```diff
@@ -111,15 +97,9 @@ conceptualContent:
         - url: mvc/views/razor.md
           itemType: concept
           text: "Razor syntax"
```

Comment:
This is Razor views as used in MVC apps, not in Razor pages apps as the card title promises.

Reply:
The Razor syntax page is for Razor syntax anywhere you use it in both "ASP.NET Core MVC or Razor Pages app".  So it mentions both app types in the doc.  The doc intro could be more clear however.